<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/img/编辑器.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="CodeEditor" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_A_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_U"/>
    <addaction name="action_Recover"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Cut"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_Replace"/>
    <addaction name="separator"/>
    <addaction name="action_All"/>
   </widget>
   <widget class="QMenu" name="menu_Q">
    <property name="title">
     <string>格式(&amp;Q)</string>
    </property>
    <addaction name="action_Wrap"/>
    <addaction name="action_Line"/>
    <addaction name="action_Font"/>
    <addaction name="action_Font_Color"/>
    <addaction name="separator"/>
    <addaction name="action_Font_Background_Color"/>
    <addaction name="action_Edit_Background_Color"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="action_Tool"/>
    <addaction name="action_State"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_Q"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_A_Save"/>
   <addaction name="separator"/>
   <addaction name="action_U"/>
   <addaction name="action_Recover"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Find"/>
   <addaction name="action_Replace"/>
   <addaction name="separator"/>
   <addaction name="action_Wrap"/>
   <addaction name="action_Line"/>
   <addaction name="action_Font"/>
   <addaction name="action_Font_Color"/>
   <addaction name="separator"/>
   <addaction name="action_Tool"/>
   <addaction name="action_State"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/新建.png</normaloff>:/img/新建.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/打开.png</normaloff>:/img/打开.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/保存.png</normaloff>:/img/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A_Save">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/另存为.png</normaloff>:/img/另存为.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="action_U">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/撤销.png</normaloff>:/img/撤销.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Recover">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/恢复.png</normaloff>:/img/恢复.png</iconset>
   </property>
   <property name="text">
    <string>恢复()</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/剪切.png</normaloff>:/img/剪切.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/复制.png</normaloff>:/img/复制.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/粘贴.png</normaloff>:/img/粘贴.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/查找.png</normaloff>:/img/查找.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/替换.png</normaloff>:/img/替换.png</iconset>
   </property>
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_All">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/自动换行.png</normaloff>:/img/自动换行.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="action_Line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/显示行号.png</normaloff>:/img/显示行号.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/字体大小.png</normaloff>:/img/字体大小.png</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="action_Font_Color">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/字体颜色.png</normaloff>:/img/字体颜色.png</iconset>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="action_Font_Background_Color">
   <property name="text">
    <string>字体背景色</string>
   </property>
  </action>
  <action name="action_Edit_Background_Color">
   <property name="text">
    <string>编辑器背景色</string>
   </property>
  </action>
  <action name="action_Tool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/工具栏.png</normaloff>:/img/工具栏.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(&amp;T)</string>
   </property>
  </action>
  <action name="action_State">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/状态.png</normaloff>:/img/状态.png</iconset>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="la2_res.qrc">
     <normaloff>:/img/关于 (1).png</normaloff>:/img/关于 (1).png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="la2_res.qrc"/>
 </resources>
 <connections/>
</ui>
